src/Admin/AbstractAdmin.php

```diff
@@ -52,9 +52,10 @@
 use Symfony\Component\Routing\Generator\UrlGeneratorInterface as RoutingUrlGeneratorInterface;
 use Symfony\Component\Security\Acl\Model\DomainObjectInterface;
 use Symfony\Component\Security\Core\Exception\AccessDeniedException;
-use Symfony\Component\Translation\TranslatorInterface;
+use Symfony\Component\Translation\TranslatorInterface as LegacyTranslationInterface;
 use Symfony\Component\Validator\Mapping\GenericMetadata;
 use Symfony\Component\Validator\Validator\ValidatorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
```

Comment:
Isnt this available on sf4.4?

Reply:
yes it is

Reply:
So in master the compat code can be droped safely right?

Reply:
Yes indeed. On master it can be typehinted with `Symfony\Contracts\Translation\TranslatorInterface` and we can drop all usages of the old legacy interface

Reply:
But actually on master most usages should be fully removed anyway :blush: 

See https://github.com/sonata-project/SonataAdminBundle/pull/6342/files#diff-61e6d1f8fafe87b868adbe28ce0ba8f6R311